packages/ksf-login/src/KSF/Login/Facebook/Sdk.js

```diff
@@ -45,7 +45,7 @@ exports._api = function(fb, path, method, params, callback) {
 exports._init = function(callback) {
   return function(fbConfig) {
     return function() { // returns an effect
-      if (window.isFBLoaded) {
+      if (window.isFBLoaded || typeof FB === 'object') {
```

Comment:
This will evaluate to `true` if `FB` is `null` (`typeof null === "object"`). It probably doesn't matter since `FB` would be either `undefined` (when not loaded) or an actual object (when loaded), but it also might become a very gnarly bug.

Let's at least check for presence of `FB` (`FB && typeof FB === 'object'`). Maybe `FB` has some field that we can check for to find out its status?

Also, how about doing `window.FB` everywhere so that it's visible that its global?